scripts/functions.sh

```diff
@@ -123,7 +123,7 @@ function getSubnets() {
   yaml="${MANIFEST_OUTPUT_PATH}/${CLUSTER_CONFIG}"
   echo "$installConfig" >${yaml}
 
-  machines=$(yq e -oj '.networking.machineNetwork[] | select(has("cidr")).cidr' "$yaml")
+  machines=$("$YQ_BIN" e -oj '.networking.machineNetwork[] | select(has("cidr")).cidr' "$yaml")
```

Comment:
Don't we need to check that `YQ_BIN` is well defined in the top of this script ? 

Something like:

```
if [ -z "${YQ_BIN+x}" ]; then YQ_BIN=$(which yq); fi
```

Reply:
YQ_BIN is initialized to `$(which yq)`so it can't be not set ?

Reply:
the `YQ_BIN` is defined in another file so it's just to be bulletproof

Reply:
if I will error then if its not set